Sources/NIOCore/EventLoop.swift

```diff
@@ -781,6 +781,31 @@ extension EventLoop {
     ) -> Scheduled<T> {
         self._flatScheduleTask(in: delay, file: file, line: line, task)
     }
+    
+    /// Schedule a `task` that is executed by this `EventLoop` after the given amount of time.
+    ///
+    /// - parameters:
+    ///     - delay: The delay between the end of one task and the start of the next.
```

Comment:
This seems wrong, as `flatScheduleTask` doesn't reschedule anything.

Reply:
I misunderstood the nature of this function, I will remove this overload as it's unnecessary.

Reply:
Sure, if we remove this overload then that's fine.